content/22.vaccines.md

```diff
@@ -237,6 +237,14 @@ Furthermore, the producers of Gam-COVID-Vac state that they can produce the vacc
 ### Inactivated Whole Virus Vaccines
 
 <!--Rephrase to match figure? Or rephrase figure?-->
+This type of vaccine uses a "killed" full viral particle to prime the immune system.
```

Comment:
Should we defined "killed"? I was going to but then realized I'm not totally sure what it means -- do they chop it up somehow? Denature it? I can't think of how else you'd "kill" a virus!

Reply:
It looks like viruses are commonly killed chemically which is what the next line is supposed to describe. Do you think there should be more detail here? Or perhaps I'm misunderstanding something..

Reply:
"These types of vaccines use full virus particles that have been rendered non-infectious by chemical or physical means. These virus particles are said to be inactivated but they still have the capacity to prime the immune system." I would suggest this maybe^^

You can't kill a virus from my understanding? just inactivate as they are technically not living (a whole other debate)